src/framework/application.js

```diff
@@ -1457,6 +1457,19 @@ Object.assign(pc, function () {
             }
         },
 
+        /**
+         * @private
+         * @function
+         * @name pc.Application#toggleRequestRetrying
+         * @description Enables or disables retrying of failed requests when loading assets.
+         * @param {Boolean} retry Whether to enable or disable retrying of requests.
+         */
+        toggleRequestRetrying: function (retry) {
+            for (var key in this.loader._handlers) {
+                this.loader._handlers[key].retryRequests = retry;
+            }
+        },
+
```

Comment:
Why is the toggling of retrying just a matter of setting the number of retries to 0? Why add additional API?

Reply:
And yes, I do appreciate this is private API, but still...

Reply:
I'm not sure what number of retries you are referring to. There is no global number of retries each URL is retried separately. This function is there to disable or enable retrying for all resource handlers at once. You can also disable or enable retrying per resource handler for different asset types.

Reply:
Oh right. Fair enough. I would just make this a property then. 'retryFailedRequests' maybe?

Reply:
That was my first thought as well however this property would only have a setter because getting back a value would not make sense since it would depend on all the individual flags of the resource handlers. So after talking with Dave we leaned towards a method instead because a property with just a setter is a little weird. Also I was going to rename this method to something like setRetryRequests or something like that.

Reply:
On reflection, maybe we should move this on to ResourceLoader and make it two methods `enableRetry`, `disableRetry`. We can then (in the future) add options like `maxRetries` to the enableRetry method.

Reply:
done